blog/_posts/2021-01-15-snap-announcement.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "A Snap package for ROOT on Linux"
+layout: archive
+author: James Carroll
+---
+
+There's a new experimental package format for ROOT, based on the [Snap package manager](https://snapcraft.io) from Canonical. This package can be ideal for new ROOT users, new Linux users, or people who's ROOT requirements might be entirely satisfied with an immutable container image. Take a look at the store listing at [https://snapcraft.io/root-framework](https://snapcraft.io/root-framework), where you'll find installations for some common distributions, E.G, [Ubuntu](https://snapcraft.io/install/root-framework/ubuntu), [Mint](https://snapcraft.io/install/root-framework/mint), [Debian](https://snapcraft.io/install/root-framework/debian), [Fedora](https://snapcraft.io/install/root-framework/fedora), [OpenSUSE](https://snapcraft.io/install/root-framework/opensuse), [Arch](https://snapcraft.io/install/root-framework/arch), and more!
+
+On Ubuntu, it's as simple as:
+```
+sudo snap install root-framework && root
+```
+
+You might be able to even just search for the ROOT Framework and install it in a single click!
+
+![ROOT in the Ubuntu App Store]({{ '/assets/images/Snap_Announcement_01.png' | relative_url }})
+
+This is full-fat installation of ROOT, complete with its utilities such as hadd, PyROOT via Python 3.8 (with SciPy, NumPy, Pandas and Matplotlib), and JupyROOT.
```

Comment:
full-fat -> a full-fat, hadd ->  `hadd`

Reply:
I've made sure to code-format all the references to hadd, changing one line later to remove the ambiguity I had over it.